modules/appeals_api/app/services/appeals_api/central_mail_updater.rb

```diff
@@ -6,6 +6,11 @@ class CentralMailUpdater
 
     MAX_UUIDS_PER_REQUEST = 100
 
+    ERROR_UNIDENTIFIED_MAIL = %w[
+      Unidentified Mail: We could not associate part or all of this submission with a Veteran.
+      Please verify the identifying information and resubmit.
+    ].join(' ')
```

Comment:
I took [this line](https://github.com/department-of-veterans-affairs/vets-api/blob/95389deab105a914736c00fcb49d24441d62a2ed/modules/vba_documents/app/models/vba_documents/upload_submission.rb#LL24C11-L24C11) directly from the Benefits Intake API.